### ISSUE TITLE
[ISSUE] Issue while turning on Log service (Android 12.1)

### ISSUE THREAD

**Issue by Humorist2601** [ukanth/afwall#1289]
**Describe the bug**
Hello, the new version 3.5.3 is working wonderfully on Android 12.1. All the previous bugs that I encountered on Beta 2 were fixed so kudos to that. Only one issue is remaining

Steps to recreate:
1. Go to menu > Preferences > Logs > Turn of Log Service
3. Go back in the app list and refresh by swiping down
4. App list gets stuck on retrieving

**Firewall Logs**
[IPv4rules.log](https://github.com/ukanth/afwall/files/9004749/IPv4rules.log)

**Smartphone (please complete the following information):**
 - Device: Xiaomi
 - Android OS: LOS 19.1

**Additional context**
I tried to find something in the logs using a logging app but there is no crash or error in the logs

**Comment by johncarterofmars** [comment]
I have a similar issue. When a new app is installed, I will have the icon indicating a rule needs to be added to allow the app to communicate. If I click that in the drop-down shade, AfWall will open to a blank screen. It is completely frozen. Eventually, I will get the message that the app isn't responding and I can close it. Alternatively, I can get to Settings > Apps and force close it. My logs are also enabled so I don't know if that is somehow connected to the above.

**Comment by Humorist2601** [comment]
Yeah, disabling the logs makes the crash go away. Disable the logs and then force close the app.

**Comment by rainbowp00p** [comment]
> Yeah, disabling the logs makes the crash go away. Disable the logs and then force close the app.

Thanks for figuring that out. It also solves my problem:
I had Log turned on and used "NFLOG".
On startup it would generate the "toast" messages of granting superuser privileges,
then it would take a long time, if ever, to appear in the status bar.
On starting the app via touch it would result in a black screen with repetitive dialogue to either wait or kill the app.
Plus the above stated hang when refreshing the app list via "touch-move-down", where i killed it via a long "back" press.

Dont know whether this could also be related to my prior update from Magisk 23 to 25.1.

Device: Sony Xperia10
Android OS: LOS 19.1-kirin (Android 12L)
Root: Magisk 25.1

Edit: Might not be related, but the "Intent Filter Verification Service" is dying alot since today.

**Comment by ukanth** [comment]
Make sure you use systemwide busybox to make it work properly.
